src/test/garfile/responsibleperson/get.controller.test.js

```diff
@@ -59,7 +59,7 @@ describe('GAR Responsible Person Get Controller', () => {
       responsibleAddressLine2: 'United Federation of Planets',
       responsibleTown: 'Alpha Quadrant',
       responsiblePostcode: 'NCC-1701D',
-      responsibleCounty: 'Earth',
+      responsibleCounty: 'GBR',
```

Comment:
should this be `responsibleCountry` ?

Reply:
From a testing perspective, no, as
* the`src/app/garfile/responsibleperson/get.controller.js` and `src/app/garfile/responsibleperson/index.njk` use `responsibleCounty` instead of `responsibleCountry` (see attached).
* However, I am happy to make the change in the `data-access-api` and elsewhere if wanted.
  - `responsibleCounty` should be `responsibleCountry`.

Reply:
ah I see, it's everywhere.. ! 😱 does it go as far as the database?